cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -14,6 +14,11 @@
   the License.
   -->
 <configuration>
+    <property>
```

Comment:
It seems to be standalone specific only. This property is required to simplify fully-qualified paths usage in config files.